kolibri/plugins/management/assets/src/vue/class-enroll-page/index.vue

```diff
@@ -1,30 +1,357 @@
 <template>
 
   <div>
-    <p>enroll for class {{ this.classId}}</p>
+    <div class="top-buttons pure-g">
+
+      <div :class="windowSize.breakpoint > 2 ? 'pure-u-1-2' : 'pure-u-1-1 align-center'">
+        <router-link :to="editClassLink">
+          <icon-button
+            :text="$tr('backToClassDetails')"
+            :primary="false">
+            <mat-svg category="navigation" name="arrow_back"/>
+          </icon-button>
+        </router-link>
+      </div>
+
+      <div :class="windowSize.breakpoint > 2 ? 'pure-u-1-2 align-right' : 'pure-u-1-1 align-center'">
+        <icon-button
+          :text="$tr('enrollSelectedUsers')"
+          :primary="true"
+          @click="openConfirmEnrollmentModal"
+          :disabled="selectedUsers.length === 0">
+          <mat-svg category="navigation" name="check"/>
+        </icon-button>
```

Comment:
@christianmemije I don't see this happening with the modals I'm implementing in `sign-in` and 'sign-up`; I get this error: 
![image](https://cloud.githubusercontent.com/assets/9877852/23732230/bcb50098-0426-11e7-988f-34b09739aabe.png)
but only if I'm trying to use the modals while in "mobile" mode using chrome.